_episodes/14-checkout.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Checkout Process"
+teaching: 15
+exercises: 15
+questions:
+- "What do I need to do to finish certifying as a Carpentries Instructor?"
+objectives:
+- "Describe the final steps required to qualify as an Instructor."
+- "Schedule your community discussion session."
+keypoints:
+- "To certify, you must contribute to a lesson, take part in a discussion, and do a teaching demo within 90 days of your training event."
+---
+
+In this short episode, we will take a moment to review the actions you will
+need to take after this training to become a fully certified Carpentries Instructor.  
+
+
+## Instructor Checkout
+
+After completing the Instructor Training workshop, there are three final steps to complete before qualifying as an Instructor. 
+Briefly, the three steps are:
+
+1.  [Make (and send us a link to) a small contribution to a lesson or glossary]({{ page.root }}/14-checkout.md/#1-lesson-contribution).
+2.  [Take part in an online community discussion session]({{ page.root }}/14-checkout.md/#2-community-discussion).
+3.  [Take part in an online teaching demonstration session]({{ page.root }}/14-checkout.md/#3-teaching-demonstration).
+
+
+All trainees have 3 months (90 days) from the end date of your training to complete checkout. If you need more time,
+3-month extensions may be requested by emailing [checkout@carpentries.org](mailto: checkout@carpentries.org). 
+Extensions may be granted for any reason up to 1 year from your training date.
+
+
+> ## Be The Expert: Checkout Q & A
+>
+> In small groups, read and discuss one of the three checkout procedures described on this page. (https://carpentries.github.io/instructor-training/20-checkout/index.html if you are viewing this text elsewhere)
+> Make notes in the Etherpad:
+> - What points do you think it is most important or helpful for people to remember?
+> - What questions or points of confusion do you have, or think others might have? 
+> When you are done, report back to the full group about that stage of the process.
+>
+> This exercise should take about 10 minutes.
+{: .challenge}
+
+## 1. Lesson Contributions
+
+### Keep It Simple!
+
+**The purpose of the lesson contribution is to demonstrate that you are prepared to draw upon your experience teaching Carpentries lessons to
+contribute towards their maintenance and improvement in the future.** For this reason, we suggest starting small! Consider commenting
+on an existing issue, fixing a typo if you find one, or contributing a single term to our multilingual glossary. If you would like
+to help us out with something more substantial, consider selecting an item from our [Help Wanted page][help-wanted]. 
+
+### What is an Issue? GitHub 101
+The curriculum for this workshop, like all Carpentries curricula, is hosted on GitHub. This gives you access to a 'back room' where the code for the 
```

Comment:
Should we include a line or two about how to find the github page for a lesson (the "edit this lesson" link) a screenshot might also be helpful